### PR TITLE
fix(infiniteGrid): fix testcase infiniteGrid for IE8

### PR DESCRIPTION
**Pull request by sculove**
## Issue
ref #121

## Details
fix testcases bug for IE8

## Preferred reviewers
@happyhj

### CODE REVIEW

File: test/unit/js/infiniteGrid.test.js [hunk 1]
```diff
@@ -90,8 +90,8 @@ test("release event handler after destroy", function(assert) {
 
 	// Then
 	setTimeout(function() {
-		equal(scrollCount, 1, "should exist scroll event");
-		equal(resizeCount, 1, "should exist resize event");
+		ok(scrollCount > 0, "should exist scroll event");
```
Comment (jongmoon) [inline test/unit/js/infiniteGrid.test.js]: Why is scrollCount changed to range check (> 0) from value check(==1) although triggered once. Is it relative with IE8?

Comment (sculove) [inline test/unit/js/infiniteGrid.test.js]: IE8 trigger double resize events.

Comment (jongmoon) [inline test/unit/js/infiniteGrid.test.js]: It would be better if it has some comments.

// resize event is fired twice in IE8

How about you?

Comment (sculove) [inline test/unit/js/infiniteGrid.test.js]: ok!

**Comment by jongmoon** [comment]
LGTM

**Comment by happyhj** [comment]
LGTM

**Pull request merged**